cli/inspect/inspector.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/alibaba/pouch/pkg/utils/templates"
 
+	"fmt"
```

Comment:
Please move this built package fmt to line L6 of this file and gofmt this file, since we have made a rule of 

> When importing packages, to improve readabilities, we should import package by sequence: system packages, project's own packages and third-party packages. And we should keep a blank line among these three kinds of packages;

In https://github.com/alibaba/pouch/blob/master/docs/contributions/code_styles.md#additional-style-rules

Reply:
omg, I will solve it.
Thanks for your reminder.